src/common/components/MenuItem.jsx

```diff
@@ -0,0 +1,22 @@
+import makeStyles from '@mui/styles/makeStyles';
+import { ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+const MenuItem = ({
+  title, link, icon, selected,
+}) => {
+  const classes = makeStyles({
+    menuItemText: {
+      whiteSpace: 'nowrap',
+    },
+  });
```

Comment:
Can you please move this outside for consistency. Similar to how we have it in other places.